Source/IntegrationTests/TestFiles/LitTests/LitTest/proof-obligation-desc/ensures-stronger.dfy.expect

```diff
@@ -0,0 +1,6 @@
+﻿ensures-stronger.dfy(16,2): Error: the method must provide an equal or more detailed postcondition than in its parent trait
+ Asserted expression: P(res') ==> b' ==> Q(res')
```

Comment:
This is a nice case where I think seeing the asserted expression is especially helpful.